docs/api/api_docs.yaml

```
@@ -2027,91 +2043,220 @@ paths:
     get:
       responses:
         '200':
-          description: OK
+          description: Retrieved replication configurations successfully
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/All-replications'
+        '404':
+          $ref: '#/components/responses/Not-found'
       tags:
         - Admin
+      summary: Get all replication configurations
+      description: |-
+        This will retrieve all the defined replication configs as a key map.
```

Not just configs - returns things like "state". A map is also just a regular JSON object. I'm not sure it's worth calling out specifically.

```suggestion
        This will retrieve all database replication definitions.
```